http-core/src/main/java/org/apache/pekko/http/javadsl/model/StatusCodes.java

```diff
@@ -97,10 +97,19 @@ private StatusCodes() {}
       org.apache.pekko.http.scaladsl.model.StatusCodes.LengthRequired();
   public static final StatusCode PRECONDITION_FAILED =
       org.apache.pekko.http.scaladsl.model.StatusCodes.PreconditionFailed();
+
+  public static final StatusCode CONTENT_TOO_LARGE =
+      org.apache.pekko.http.scaladsl.model.StatusCodes.ContentTooLarge();
+
+  /** @deprecated deprecated in favor of CONTENT_TOO_LARGE since 1.0.0 */
```

Comment:
this should be `since 1.1.0` - 1.0.0 was released without this change

Reply:
:+1: good point